find.lua

```diff
@@ -0,0 +1,461 @@
+local ffi = require 'ffi'
+
+find = {}
+find.__index = find
+
+local function call(layer, f, ...)
+   local status = cudnn.call(f, ...)
+   if status ~= ffi.C.CUDNN_STATUS_SUCCESS and cudnn.verbose then
+      local pad = ffi.new('int[5]')
```

Comment:
this block makes no sense to me. Once status is ~= ffi.C.CUDNN_STATUS_SUCCESS, why are you calling cudnnGetConvolutionNdDescriptor, but not actually using any of the resulting tensors?